controller.go

```diff
@@ -65,6 +65,7 @@ var (
 	firstRun                   bool = true
 	cwAlarmConfigMap           string
 	cwAlarmConfigMapLocation   *kubernetes.ResourceLocation
+	loadBalancerType           string
```

Comment:
Probably makes sense to use kingpin for flag handling at this point as it can do a lot of that out of the box. I would do that in a separate PR though. WDYT?